app/src/components/features/rules/TestThisRule/index.ts

```diff
@@ -1,2 +1,3 @@
 export { TestThisRule } from "./TestThisRule";
-export * from "./helpers";
+export * from "./utils/testReports";
+export * from "./utils/widget";
```

Comment:
These 2 exports are not components but actions. Should we export helper actions from the component index?